test/extensions/matching/input_matchers/cel_matcher/cel_matcher_test.h

```diff
@@ -0,0 +1,630 @@
+        value: 45
+      }
+      positions {
+        key: 10
+        value: 47
+      }
+      positions {
+        key: 11
+        value: 20
+      }
+    }
+)pb";
+
+// Compiled CEL expression string: response.trailers['transfer-encoding']=='chunked'
+inline constexpr char ReponseTrailerCelExprString[] = R"pb(
+  expr {
+    id: 5
+    call_expr {
+      function: "_==_"
+      args {
+        id: 3
+        call_expr {
+          function: "_[_]"
+          args {
+            id: 2
+            select_expr {
+              operand {
+                id: 1
+                ident_expr {
+                  name: "response"
+                }
+              }
+              field: "trailers"
+            }
+          }
+          args {
+            id: 4
+            const_expr {
+              string_value: "transfer-encoding"
+            }
+          }
+        }
+      }
+      args {
+        id: 6
+        const_expr {
+          string_value: "chunked"
+        }
+      }
+    }
+  }
+  source_info {
+    location: "<input>"
+    line_offsets: 50
+    positions {
+      key: 1
+      value: 0
+    }
+    positions {
+      key: 2
+      value: 8
+    }
+    positions {
+      key: 3
+      value: 17
+    }
+    positions {
+      key: 4
+      value: 18
+    }
+    positions {
+      key: 5
+      value: 38
+    }
+    positions {
+      key: 6
+      value: 40
+    }
+  }
+)pb";
+
+// Compiled CEL expression string: request.path == '/foo' || request.headers['user'] ==
+// 'staging'.
+inline constexpr char RequestHeaderOrPathCelString[] = R"pb(
+  expr {
+    id: 11
+    call_expr {
+      function: "_||_"
+      args {
+        id: 3
+        call_expr {
+          function: "_==_"
+          args {
+            id: 2
+            select_expr {
+              operand {
+                id: 1
+                ident_expr {
+                  name: "request"
+                }
+              }
+              field: "path"
+            }
+          }
+          args {
+            id: 4
+            const_expr {
+              string_value: "/foo"
+            }
+          }
+        }
+      }
+      args {
+        id: 9
+        call_expr {
+          function: "_==_"
+          args {
+            id: 7
+            call_expr {
+              function: "_[_]"
+              args {
+                id: 6
+                select_expr {
+                  operand {
+                    id: 5
+                    ident_expr {
+                      name: "request"
+                    }
+                  }
+                  field: "headers"
+                }
+              }
+              args {
+                id: 8
+                const_expr {
+                  string_value: "user"
+                }
+              }
+            }
+          }
+          args {
+            id: 10
+            const_expr {
+              string_value: "staging"
+            }
+          }
+        }
+      }
+    }
+  }
+  source_info {
```

Comment:
Same, source_info is not needed in a test. You can also erase "id" fields if you want.